web/frontend/src/components/input-controller/index.svelte

```diff
@@ -61,7 +62,7 @@
   })(status.eventsList);
 </script>
 
-<v-collapse title={name}>
+<Collapse title={name}>
```

Comment:
Why not in Prime?

Reply:
Will be backported, but has new optimizations